src/e2e/utils/helpers.ts

```diff
@@ -45,6 +45,28 @@ export const getBaseUrl = () => {
   return ENV_URLS[process.env.E2E_TEST_ENV as ENV] || ENV_URLS.local;
 };
 
+export const delay = async (time: number) => {
+  return new Promise(function (resolve) {
+    setTimeout(resolve, time);
+  });
+};
+
+export const waitForUrlOrTimeout = async (
+  page: Page,
+  urlSubstring: string,
+  timeout: number,
+) => {
+  const tries = 0;
```

Comment:
Can you explain this bit?
I see this is a `const` and we check if `tries < 10` below on L61, but... we never seem to increment `tries` on each reattempt?

Reply:
ah updating to let, test been accidentally using the timeout comparison